_inc/lib/core-api/class.jetpack-core-api-xmlrpc-consumer-endpoint.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This is the base class for every Core API endpoint Jetpack uses.
+ *
+ */
+abstract class Jetpack_Core_API_XMLRPC_Consumer_Endpoint {
+
+	/**
+	 * An instance of the Jetpack XMLRPC client to make WordPress.com requests
+	 *
+	 * @private
+	 * @var Jetpack_IXR_Client
+	 */
+	private $xmlrpc;
```

Comment:
Yes, I had the same question, but I was satisfied by the fact that we have the class historically named so. And the variable is called `xmlrpc` because it is a client to make XMLRPC calls. Do you think this variable should be renamed to represent its origin rather than its purpose?